db/projects/chatcoin/project.yaml

```diff
@@ -0,0 +1,23 @@
+name: ChatCoin
+web:
+  - 'http://www.openchat.co/'
+ledgers:
+  - id: chatcoin
+    name: ChatCoin
+    type: blockchain
+    networks:
+      - id: main
```

Comment:
Add type: main

Reply:
You have commented the very first comment. In the fact, ChatCoin does not have network since it's Ethereum token